src/Handler/CompositeImportErrorHandler.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Brille24\SyliusCustomerOptionsPlugin\Handler;
+
+use Webmozart\Assert\Assert;
+
+class CompositeImportErrorHandler implements ImportErrorHandlerInterface
```

Comment:
Do we need to have such an elaborate error handling as we will have an error handler in our implementation.